src/main/java/Duke.java

```diff
@@ -1,10 +1,343 @@
+import duke.exception.IllegalCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String INVALID_COMMAND_PRINTER = "One hour of lifespan has been deducted, in accordance with our Terms and Services.";
+    private static final String VALID_COMMAND_PRINTER = "Command acknowledged. Reducing user lifespan by 30 minutes.";
+    private static final String LOGO =
+              "    // | |     //   ) )  // | |  \\\\ / / \\\\    / / //   ) )\n"
+            + "   //__| |    //___/ /  //__| |   \\  /   \\\\  / / ((\n"
+            + "  / ___  |   / ___ (   / ___  |   / /     \\\\/ /    \\\\\n"
+            + " //    | |  //   | |  //    | |  / /\\\\     / /       ) )\n"
+            + "//     | | //    | | //     | | / /  \\\\   / / ((___ / /";
+    private static String command;
+    private static ArrayList<Task>tasks = new ArrayList<>();
```

Comment:
Try to have spaces for easier readability. Like `ArrayList<Task> tasks`. Throughout your code you also seem to squish everything together which makes it hard to read. The use of spaces would help distinguish the different objects that you are working with and makes it easier to identify when you are looking back at your code.